isotovideo

```diff
@@ -435,6 +440,8 @@ while ($loop) {
         if ($rsp->{cmd} eq 'version') {
             my $result = {test_git_hash => $test_git_hash, version => $INTERFACE};
             myjsonrpc::send_json($r, $result);
+            $result = {needles_git_hash => $needles_git_hash, version => $INTERFACE};
+            myjsonrpc::send_json($r, $result);
```

Comment:
question is if someone cares. The version call itself is pretty unused - but just adding the hash to the result would be good enough *if* someone cared